portfolio/data-servlet/main/webapp/index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
```

Comment:
It looks like we have some duplicate files in the following paths:
* portfolio/src/main/webapp/*
* portfolio/data-servlet/main/webapp/*

Do we need the files in `portfolio/data-servlet/main/webapp`?